app/api/orders.py

```diff
@@ -86,6 +87,16 @@ def before_create_object(self, data, view_kwargs):
                 raise ConflictException({'pointer': '/data/relationships/attendees'},
                                         "Attendee with id {} does not exists".format(str(ticket_holder)))
 
+        ticket = db.session.query(Ticket).filter_by(
+            id=int(ticket_holder_object.ticket_id), deleted_at=None
+        ).first()
```

Comment:
This only checks for last ticket holder. Please test your PRs before asking for reviews